options.go

```diff
@@ -30,6 +30,13 @@ const (
 	HintBPTSparseIdxMode
 )
 
+type ExpiredDeleteType uint8
+
+const (
+	TimeWheel ExpiredDeleteType = iota
```

Comment:
Please add more comments to elaborate more details. That will not confuse the code reader.